model/authorization/TestTakerAuthorizationService.php

```diff
@@ -104,6 +113,34 @@ public function isProctored($deliveryId, User $user)
             : $deliverySyncService->isProctoredByDefault();
     }
 
+    /**
+     * @param string $deliveryId
+     * @param string $state
+     * @return bool
+     * @throws common_Exception
+     */
+    public function isActiveUnSecureDelivery($deliveryId, $state)
+    {
+        return $this->isNotSecure($deliveryId) && $state === DeliveryExecutionInterface::STATE_ACTIVE;
+    }
+
+    /**
+     * @param string $deliveryId
+     * @return bool
+     * @throws common_Exception
+     */
+    private function isNotSecure($deliveryId)
```

Comment:
Could you please rename it to `isSecure` or `isSecureDelivery` and use `!isSecure()` in a place where you call this method.

Reply:
changed